lib/commands/context.js

```diff
@@ -192,6 +194,12 @@ async function setupExistingChromedriver (chromedriver) {
 }
 
 async function setupNewChromedriver (opts, curDeviceId, adbPort) {
+  // if a port wasn't given, pick a random available one
+  if (opts.chromeDriverPort === null || opts.chromeDriverPort === undefined) {
```

Comment:
Note: `lint` Complained about using `x == null` here, despite it seeming to be the consensus for checking presence of a variable:
http://stackoverflow.com/a/2647888